### ISSUE TITLE
"You may only use the Pay-fare option after completing Pirate's Treasure"

### ISSUE THREAD

**Issue by orange451** [itsdax/Runescape-Web-Walker-Engine#53]
Just creating this ticket to let you know that the walker always attempts to click the "Pay" option (when traveling to Karamja) even when you havent completed Pirate's Treasure quest. Not a super high priority, because the dialog still pops up. Though it may look a little odd on Jagex's end if they see a player constantly doing something they can't do?

**Comment by FALSkills** [comment]
It shouldn't really matter, this behavior isn't odd at all and picking the "proper" choice every time seems more bot-like to me.  The vast majority of fast travel options in-game do not behave in the same way as the Port Sarim pay-fare, and so even regular players who have not completed it would be apt to choose the "Pay-fare" option regardless of their quest completion status.  The resulting NPC conversation is the same anyway.

**Comment by FALSkills** [comment]
tl;dr this does not need to be changed. Feel free to change it locally if you think it does.
